server/.golangci.yml

```diff
@@ -57,3 +56,7 @@ issues:
     - linters:
       - staticcheck
       text: SA1019
+
+    - linters:
+        - errcheck
+      path: "channels|platform|cmd/mattermost|public/model|public/plugin|public/shared/mlog|enterprise/data_retention|enterprise/saml|enterprise/cloud|enterprise/ldap"
```

Comment:
The ignore list is quite broad. We can scope it down more granularly once we have a clear idea of how we do that. (Campain, staff member, don't work on it for now?)

First, I want to fix the issues in the enterprise code, as we can't do that with the community.